corehq/apps/es/cases.py

```diff
@@ -21,6 +21,9 @@
 
 class CaseES(HQESQuery):
     index = 'cases'
+    default_filters = {
+        'is_commcare_case': filters.doc_type("CommCareCase"),
```

Comment:
see my other comment, but i would have thought deleted cases would be removed from the index